src/ODBC/OdbcConnection.php

```diff
@@ -84,11 +86,17 @@ public function quoteIdentifier(string $str): string
     protected function doQuery(string $query): QueryResult
     {
         try {
-            $stmt = odbc_exec($this->connection, $query);
+            /** @var resource|false $stmt */
+            $stmt = @odbc_exec($this->connection, $query);
         } catch (Throwable $e) {
             throw new OdbcException($e->getMessage(), $e->getCode(), $e);
         }
 
+        // "odbc_exec" can generate warning, if "set_error_handler" is not set, so we are checking it manually
+        if ($stmt === false) {
```

Comment:
Zistil som, ze `odbc_*` generuju warning a vracaju `false` v pripade chyby.
Na exception sa to prevadza, iba ak je nastaveny `set_error_handler`, ... preto som to radsej v kode upravil a kontrolujem to manualne. Aby sme mali istotu, ze za kazdych okolnosti to bude exception.